openedx/features/effort_estimation/block_transformers.py

```diff
@@ -0,0 +1,225 @@
+"""
+Effort Estimation Transformer implementation.
+Adds effort estimations for block types it recognizes.
+"""
+
+import math
+
+import crum
+import lxml.html
+from django.utils.functional import cached_property
+from edxval.api import get_videos_for_course
+
+from openedx.core.djangoapps.content.block_structure.transformer import BlockStructureTransformer
+from openedx.core.lib.mobile_utils import is_request_from_mobile_app
+
+from .toggles import EFFORT_ESTIMATION_LOCATION_FLAG
+
+
+class EffortEstimationTransformer(BlockStructureTransformer):
+    """
+    A transformer that adds a partition condition for all graded content
+    so that the content is only visible to verified users.
+
+    This transformer requires that the UserPartitionTransformer also be included in your transformer list.
+    """
+    WRITE_VERSION = 1
+    READ_VERSION = 1
+
+    # Public xblock field names
+    EFFORT_ACTIVITIES = 'effort_activities'
```

Comment:
these are set to effort_activities and effort_time here, but elsewhere in your PR you're using those hardcoded values.  
Should those other spots be importing from here?

Reply:
I used these class properties in a couple more places where we already had a reference to this class. But in general, I'd like to avoid importing this random transformer just to grab a string from it. The string can't really change anyway.